src/test/java/seedu/guest/commons/util/ConfigUtilTest.java

```diff
@@ -64,7 +64,7 @@ public void read_extraValuesInFile_extraValuesIgnored() throws DataConversionExc
     private Config getTypicalConfig() {
         Config config = new Config();
         config.setLogLevel(Level.INFO);
-        config.setUserPrefsFilePath(Paths.get("preferences.json"));
+        config.setGuestPrefsFilePath(Paths.get("preferences.json"));
```

Comment:
Similar comment to the above about the renaming of class names and variables, should we revert back to the original?